src/bumerang/app.py

```diff
@@ -35,6 +37,9 @@ def __init__(self):
             (r'/profile/([0-9]+)/?', ProfileHandler),
             (r'/request/?', BorrowHandler),
             (r'/request/([0-9]+)/?', BorrowHandler),
+            (r'/requests/recent/?', BorrowsByRecentHandler),
```

Comment:
I'm not 100%, but do we need this end point, if we have the one on the next line?